crates/uv-cli/src/lib.rs

```diff
@@ -624,7 +624,15 @@ pub enum PipCommand {
         after_long_help = ""
     )]
     Compile(PipCompileArgs),
-    /// Sync an environment with a `requirements.txt` file.
+    /// Sync an environment with a `requirements.txt` file
```

Comment:
Can we keep the trailing period? :)

Reply:
It's trimmed on display by Clap, I think.